spec/std/ecr/ecr_lexer_spec.cr

```diff
@@ -6,47 +6,47 @@ describe "ECR::Lexer" do
     lexer = ECR::Lexer.new("hello")
 
     token = lexer.next_token
-    token.type.should eq(:STRING)
+    token.type.should eq(ECR::Lexer::Token::Type::String)
```

Comment:
Yeah, I'd like something like that but we need to think how to implement it.

Reply:
just `be(& : T -> Bool)`??

Reply:
Sorry, I don't understand how would that work. Also with `&` it can't work, it'll need to capture the block, but there's no way to figure out what the `T` is.

Reply:
ohhh, yeah, you're right...

Reply:
I ended up doing a trick used by @bcardiff in one of his PRs: use a file-private method to convert a symbol to enum. I think it's a good workaround, and usually you only need this in a spec file.